### PR TITLE
Current code

### PR DESCRIPTION
**Pull request by CristobalMe**

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const cors = require('cors');
+const morgan = require('morgan');
+const express = require('express');
+const SequelizeStoreInit = require('connect-session-sequelize');
+const session = require('express-session');
+const Sequelize = require('sequelize');
```
Comment (toriqyanM) [inline backend/server.js:6]: Sequelize is another ORM, similar to Prisma. If you choose to use Prisma here, then there is no need to include sequelize.

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -1,22 +1,58 @@
...
+import { UserContext } from './UserContext';
+
+
 function App() {
 
+  const [user, setUser] = useState(() => {
+    // Retrieve the user data from storage or set it to null if not found
+    const storedUser = localStorage.getItem('user');
+    // return storedUser ? JSON.parse(storedUser) : null;
+    return null
```
Comment (toriqyanM) [inline frontend/src/App.jsx:23]: This is where user is initialized in your state. The logic should be search in cookie, if it does exist in cookie, retrieve from cookie. Why is it returning null here?

Comment (CristobalMe) [reply frontend/src/App.jsx:23]: It was for debugging purposes

File: backend/server.js (hunk 1 above)
Comment (CristobalMe) [reply backend/server.js:6]: To Do:
Delete sequelize db & pass content to prisma

**Pull request closed without merging**